easel/lib/models/nft_format.dart

```diff
@@ -27,12 +35,14 @@ class NftFormat {
         NftFormat(
           format: NFTTypes.image,
           extensions: ['jpg', 'png', 'svg', 'heif', 'jpeg', 'gif'],
+          extensionsUI: ['jpg', 'png', 'svg', 'heic',],
```

Comment:
Revert.

Reply:
In the updated figma some extensions are removed so I removed those extensions but you asked for adding them again so I created a new extension list so the UI matches the Figma.

<img width="423" alt="image" src="https://user-images.githubusercontent.com/115459890/206748147-5cde251d-70fa-4578-9190-6c5ea8f82644.png">

Reply:
> In the updated figma some extensions are removed so I removed those extensions but you asked for adding them again so I created a new extension list so the UI matches the Figma.
> 
> 
> 
> <img width="423" alt="image" src="https://user-images.githubusercontent.com/115459890/206748147-5cde251d-70fa-4578-9190-6c5ea8f82644.png">
> 
> 
> 
> 

Extension list should remain untouched from how it is now in-app

![image](https://user-images.githubusercontent.com/74597290/206750221-0dfe1209-5bc0-404e-bdc4-63773ae5dc15.png)

Reply:
ok I am reverting my changes regarding extension